plugins/Hashes/plugin.py

```diff
@@ -111,16 +111,15 @@ def algorithms(self, irc, msg, args):
         """<takes no arguments>
 
         Returns the list of available algorithms."""
-        algos = list(hashlib.algorithms_available)
-        irc.reply("%s" % algos.join(', '))
+        irc.reply("%L" % hashlib.algorithms_available)
```

Comment:
No, `format("%L", hashlib.algorithms_available)`. You need to use Limnoria's format function from supybot.utils.str